curriculum/challenges/chinese-traditional/13-relational-databases/learn-relational-databases/learn-github-by-building-a-list-of-inspirational-quotes.md

```diff
@@ -3,7 +3,7 @@ id: 602da04222201c65d2a019f3
 title: Learn GitHub by Building a List of Inspirational Quotes
 challengeType: 12
 helpCategory: Relational Databases
-url: https://github.com/moT01/.learn-github-by-building-a-list-of-inspirational-quotes
+url: https://github.com/freeCodeCamp/learn-github-by-building-a-list-of-inspirational-quotes
```

Comment:
Just to point out - this link leads to 404 error

Reply:
thanks @gikf - yea, I probably could have left that one for now.